web/template/layout/search_filter.gohtml

```diff
@@ -1,5 +1,5 @@
 {{ define "search-filter" }}
-    <div class="moj-filter-layout__filter">
+    <div class="moj-filter-layout__filter {{ if not .Filters.Set }}govuk-!-display-none{{ end }}">
```

Comment:
The behaviour should now be that the filter panel is hidden by default, but shown if filters are selected

Reply:
I added [a comment](https://github.com/ministryofjustice/moj-frontend/discussions/197) to the moj-frontend discussion so this might be considered in future changes to that component.

Reply:
Nice one, thankyou! Hopefully so yeah :)